src/data/dapp_promotions.json

```diff
@@ -1,4 +1,10 @@
 [
+  {
+    "name": "Lucky - No-loss lottery",
+    "shortDescription": "The more you stake, the more chance you have to win a raffle!",
+    "link": "https://portal.astar.network/astar/dapp-staking/dapp?dapp=zsv1gvepvmwfdshmwgczs4zyvmmwesbjwqjn4wdpuefrrpy",
+    "img": "images/dapp_promotions/lucky.jpg"
```

Comment:
Please move image to IPFS or some other provide

Reply:
Done